app/views/devise/registrations/new.html.erb

```diff
@@ -14,11 +14,8 @@
   </div>
 
   <div class="field">
-    <%= f.fields_for :department do |d| %>
-      <%= d.label :department %>
-      <%= d.select :name,
-          Department::DEPARTMENT, class: "department dropdown"%>
-    <% end %>
+      <%= f.label :department_id %>
+      <%= collection_select(:department_id, :department_id, Department.all, :id, :name) %>
```

Comment:
I haven't worked with rails helpers in a while, but looking up form helpers page and scrolling to select, in the [option tags from a collection of arbitrary objects section](http://guides.rubyonrails.org/form_helpers.html#option-tags-from-a-collection-of-arbitrary-objects), it looks like you'll want something like...

```ruby
<%= f.collection_select(:department_id, Department.all, :id, :name) %>
```

Does that work?